tensorzero-core/src/db/clickhouse/dataset_queries.rs

```diff
@@ -667,6 +699,60 @@ impl DatasetQueries for ClickHouseConnectionInfo {
     }
 }
 
+/// Converts a vec of OrderBy terms to the correct ClickHouse ORDER BY clauses.
+fn get_order_by_clause(
+    order_by: Option<&Vec<DatapointOrderBy>>,
+    search_query_experimental: Option<&String>,
+) -> Result<String, Error> {
+    let Some(order_by_vec) = order_by else {
+        return Ok("ORDER BY updated_at DESC, id DESC".to_string());
+    };
+    if order_by_vec.is_empty() {
+        return Ok("ORDER BY updated_at DESC, id DESC".to_string());
+    }
+
+    // Validate that if SearchRelevance is used, search_query_experimental must be present
+    for order_spec in order_by_vec {
```

Comment:
do we need to validate that there's a single search term?

Reply:
or rather, do we need to have the interface potentially handle multiple terms? given that it is experimental perhaps not worth doing rn